content/zh/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -166,24 +166,18 @@ applies a policy for setting the `phase` of all Pods on the lost node to Failed.
 如果某节点死掉或者与集群中其他节点失联，Kubernetes
 会实施一种策略，将失去的节点上运行的所有 Pod 的 `phase` 设置为 `Failed`。
 
-<!--
-## Container states
+Pod 有一个 PodStatus 对象，其中包含一个 [PodCondition](/docs/resources-reference/v1.7/#podcondition-v1-core) 数组。PodCondition 数组的每个元素都有一个 `type` 字段和一个 `status` 字段。`type` 字段是字符串，可能的值有 PodScheduled、Ready、Initialized 和 Unschedulable。`status` 字段是一个字符串，可能的值有 True、False 和 Unknown。
```

Comment:
- Do we have English source for the translation above?
- PodCondition is not pointing to a valid location, it should be a valid, Chinese version page.